app/src/main/java/org/p2p/wallet/infrastructure/network/interceptor/RpcInterceptor.kt

```diff
@@ -30,7 +45,54 @@ class ServerErrorInterceptor(
         }
     }
 
-    private fun handleResponse(response: Response): Response {
+    private val requestBuffer = Buffer()
```

Comment:
do we really need to create requestBuffer as a field? maybe local variable is enough?